third_party/BUILD

```diff
@@ -0,0 +1,7 @@
+licenses(["notice"])  # Apache 2
+
+exports_files([
```

Comment:
potentially nicer to expose this as a filegroup and depend on that, just so you get real visibility and you just have to add new headers in 1 place

Reply:
Good thought. I'll try it out locally, though may delay on making the change, since we've made it through the CI gauntlet here.